etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -71,13 +71,13 @@ spec:
           #value:
         - name: STORAGE_BACKEND
           value: {{ include "pachyderm.storageBackend" . | quote }}
-          {{- if eq .Values.deployTarget "LOCAL" }}
+          {{- if eq (include "pachyderm.storageBackend" . ) "LOCAL" }}
         - name: STORAGE_HOST_PATH
           value: {{ .Values.pachd.storage.local.hostPath | default $randHostPath }}pachd
           {{- end }}
         - name: WORKER_IMAGE
           value: "{{ .Values.worker.image.repository }}:{{ default .Chart.AppVersion .Values.pachd.image.tag }}"
-        {{- if and (eq .Values.deployTarget "LOCAL") .Values.pachd.storage.local.requireRoot }}
+        {{- if and (eq ( include "pachyderm.storageBackend" . ) "LOCAL") .Values.pachd.storage.local.requireRoot }}
         - name: WORKER_USES_ROOT
           value: "True"
```

Comment:
Never really thought about it but it's weird that "True" is the value, and not "true".